### PR TITLE
Added 0.3.0 changes

### PR DESCRIPTION
**Pull request by jotaemepereira**
### :pushpin: References
[Trello card](https://trello.com/c/SOszSLE6/310-release-10-sdk)

### :tophat: What is the goal?

Update the changelog with the `0.3.0` version changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 Change Log
 ==========
+Version 0.3.0 *(2019-02-04)*
+----------------------------
+
+* Prefill attachments  
+
+Bug Fixes:
+
+* Ensured all places the Beacon Config is accessed it has the overrides applied
```
Comment (chrisbrookins) [inline CHANGELOG.md]: I don't follow this one - can we reword it to be less about what we did, and more about the benefit?  I would reword it but I don't know what benefit we are providing to developers.  Is it something like 'You can now override every Beacon Builder setting in the SDK by doing X, Y and Z' (the config is determined by Builder, so lets refer to that if that is what we mean)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
 ==========
+Version 0.3.0 *(2019-02-04)*
+----------------------------
+
+* Prefill attachments  
+
+Bug Fixes:
+
+* Ensured all places the Beacon Config is accessed it has the overrides applied 
+* Crash when launching BeaconActivity with Context rather than Activity
```
Comment (chrisbrookins) [inline CHANGELOG.md]: I'd prefer we just say 'Improved stability by resolving several crashes' unless these were customer reported - if customer reported is this how they raised the issue?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+----------------------------
+
+* Prefill attachments  
+
+Bug Fixes:
+
+* Ensured all places the Beacon Config is accessed it has the overrides applied 
+* Crash when launching BeaconActivity with Context rather than Activity 
+* Crash when `lastThread` is null on `ConversationsAdapter`
+* Fix colors when button has low contrast
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: Fix colors when button has low contrast
should be
Fix colors when a button has low contrast

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+* Fix message icon is white on white background
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: * Fix white message icons on white backgrounds.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
+* Fix back/navigate up button should have the correct Android design. (Use arrow_back, not keyboard_backspace)
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: * Fix back/navigate up button to have the correct Android design. (Use arrow_back, not keyboard_backspace)

**Comment by jotaemepereira** [comment]
@adrianne-helpscout @chrisbrookins I've just updated the changelog with your suggestions! I also added a link to the reported customer issues.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
 ==========
+Version 0.3.0 *(2019-02-04)*
+----------------------------
+
+* Prefill attachments 
+
+Bug Fixes:
+
+* You can now correctly override the `messageEnabled` and `docsEnabled` flags on your Beacon.
+* [#33](https://github.com/helpscout/beacon-android-sdk-sample/issues/33) Fix crash when launching BeaconActivity with Context rather than Activity.
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: Fix should be Fixed a crash

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,19 @@
...
+* [#36](https://github.com/helpscout/beacon-android-sdk-sample/issues/36) Fix crash when `lastThread` is null on `ConversationsAdapter`.
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: Fix should be Fixed a crash

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,19 @@
...
+* [#39](https://github.com/helpscout/beacon-android-sdk-sample/issues/39) Fix colors when a button has low contrast.
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: How about if we try this:
Fixed button coloring in low contrast conditions.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,19 @@
...
+* [#39](https://github.com/helpscout/beacon-android-sdk-sample/issues/39) Fix white message icons on white backgrounds.
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: Fix should be Fixed

Comment (adrianne-helpscout) [reply CHANGELOG.md]: Also, remove the period at the end of the sentence

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,19 @@
...
+* [#39](https://github.com/helpscout/beacon-android-sdk-sample/issues/39) Fix back/navigate up button to have the correct Android design. (Use arrow_back, not keyboard_backspace)
```
Comment (adrianne-helpscout) [inline CHANGELOG.md]: Fixed back/navigate-up

**Comment by jotaemepereira** [comment]
@elyse-roach @adrianne-helpscout Thanks for the feedback! Addressed the mentioned changes
